code/game/machinery/recycler.dm

```diff
@@ -152,13 +152,12 @@
 		playsound(src, item_recycle_sound, (50 + nom.len*5), TRUE, nom.len, ignore_walls = (nom.len - 10)) // As a substitute for playing 50 sounds at once.
 	if(not_eaten)
 		playsound(src, 'sound/machines/buzz-sigh.ogg', (50 + not_eaten*5), FALSE, not_eaten, ignore_walls = (not_eaten - 10)) // Ditto.
-	if(!ismob(AM0))
+
+	for (var/atom/movable/item in AM0.contents)
```

Comment:
as anything?

Reply:
IIRC as anything doesn't work on byond lists like contents

Reply:
i see
shouldnt it be in AM0 instead of in AM0.contents?

Reply:
it does last i checked